hvplot/converter.py

```diff
@@ -1371,6 +1379,23 @@ def method_wrapper(ds, x, y):
         layers = _transfer_opts_cur_backend(layers)
         return layers
 
+    def _aggregate_obj(self, operation, obj, opts):
+        def exceeds_aggregation_threshold(plot):
+            vdim = plot.vdims[0].name
+            data = plot.data[vdim]
+            data_size = np.size(data)
```

Comment:
Is this going to work on all data types supported by hvPlot? I haven't checked myself, asking you first to know whether you thought about it in which case you can just confirm and resolve this comment!

Reply:
Yes I believe it always returns a numpy array with `plot.data[vdim]`